src/main/java/org/multipleton/labs/dto/BookDto.java

```diff
@@ -0,0 +1,56 @@
+package org.multipleton.labs.dto;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class BookDto {
+
+    private Long id;
+
+    private String title;
+
+    private Long authorId;
+
+    private String tags;
```

Comment:
It's better to have a `Set` instead of just `String`, because frontend can use it in different ways. Also it's better to receive an array in `Controller`, because you don't need to think about separator and parsing. For now we can leave `String`.